--- a/lib/service/servicemp3.cpp	2014-09-07 21:01:58.399800917 +0200
+++ b/lib/service/servicemp3.cpp	2014-09-07 21:13:51.629120119 +0200
@@ -2225,7 +2225,7 @@
 #if GST_VERSION_MAJOR < 1
 				std::string line((const char*)GST_BUFFER_DATA(buffer), len);
 #else
-				std::string line(len);
+				std::string line((const char*)len);
 				gst_buffer_extract(buffer, 0, (char*)line.data(), len);
 #endif
 				eDebug("got new text subtitle @ buf_pos = %lld ns (in pts=%lld), dur=%lld: '%s' ", buf_pos, buf_pos/11111, duration_ns, line.c_str());
